src/elements/lead-container/lead-container.snapshot.spec.ts

```diff
@@ -11,19 +13,22 @@ import '../link/block-link/block-link.js';
 import '../title.js';
 import './lead-container.js';
 
+const imageUrl = import.meta.resolve('./assets/lucerne.png');
+
 describe(`sbb-lead-container`, () => {
   let element: SbbLeadContainerElement;
 
   beforeEach(async () => {
     element = await fixture(
       html`<sbb-lead-container>
-        <sbb-image slot="image"></sbb-image>
+        <sbb-image slot="image" image-src=${imageUrl}></sbb-image>
       </sbb-lead-container>`,
     );
+    await waitForImageReady(element.querySelector<SbbImageElement>('sbb-image')!);
```

Comment:
Is the generic Parameter needed? It should default to `SbbImageElement` from my understanding.